app/services/cohorts/create_schedule.rb

```
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Cohorts
+  class CreateSchedule
+    attr_reader :path_to_csv, :cohort, :klass
+
+    def initialize(path_to_csv:, cohort:, klass:)
+      @path_to_csv = path_to_csv
```

maybe we can fail early if there is no cohort/csv passed to the class?